Sources/PointFree/Routes.swift

```diff
@@ -7,142 +8,123 @@ import Prelude
 import Styleguide
 
 public let siteMiddleware: Middleware<StatusLineOpen, ResponseEnded, Prelude.Unit, Data?> =
-  requireHerokuHttps
-    <<< redirectUnrelatedDomains
+  requireHerokuHttps(allowedInsecureHosts: allowedInsecureHosts)
+    <<< redirectUnrelatedHosts(allowedHosts: allowedHosts, canonicalHost: canonicalHosts)
```

Comment:
can now use new middleware transformers that were merged https://github.com/pointfreeco/swift-web/pull/55